rfd/0116-label-expressions.md

```diff
@@ -1,6 +1,6 @@
 ---
 authors: Nic Klaassen (nic@goteleport.com)
-state: draft
+state: implemented
```

Comment:
```suggestion
state: implemented (v13.2.0)
```

Doesn't have to be exact, but gives future us a rough idea of when this was released.